src/test/java/jimmy/mcgymmy/commons/core/ConfigTest.java

```diff
@@ -16,12 +24,55 @@ public void toString_defaultObject_stringReturned() {
         assertEquals(defaultConfigAsString, new Config().toString());
     }
 
+    @Test
+    public void loggerLevelCorrect() {
```

Comment:
do we need this?

Reply:
Hmm not exactly. It's for testing if the set function works correctly.

Reply:
i meant i don't think these tests add any value. Maybe we should remove this

checking getters/setters are quite redundant to me imo

Reply:
Hmm ok
Removed